### PR TITLE
Feature/aot mode #944

### PR DESCRIPTION
**Pull request by nvolodymyr**
fixed all files whitch bloked start aot build

### CODE REVIEW

File: src/app/component/eco-news/components/remaining-count/remaining-count.component.html [hunk 1]
```diff
@@ -1,2 +1,2 @@
-<p *ngIf="[remainingCount] === 1">{{ remainingCount }} item found </p>
-<p *ngIf="[remainingCount] !== 1">{{ remainingCount }} items found</p>
+<p *ngIf="remainingCount > 0">{{ remainingCount }} item found </p>
+<p *ngIf="remainingCount < 1">{{ remainingCount }} items found</p>
```
Comment (mplesha) [inline src/app/component/eco-news/components/remaining-count/remaining-count.component.html:2]: it's not correspond to the flow what was
explain this

File: src/app/component/home/models/NewsDto.ts [hunk 2]
```diff
@@ -1,12 +1,6 @@
-export class NewsDto {
+export interface NewsDto {
   id: number;
   title: string;
   text: string;
   creationDate: string;
```
Comment (mplesha) [inline src/app/component/home/models/NewsDto.ts:5]: explain those changes

Comment (nvolodymyr) [reply src/app/component/home/models/NewsDto.ts:5]: image

Comment (mplesha) [reply src/app/component/home/models/NewsDto.ts:5]: 🙂 the question is - why you need this change?

**Comment by mplesha** [comment]
1. this
```html
<p *ngIf="remainingCount > 0">{{ remainingCount }} item found </p>
<p *ngIf="[remainingCount] !== 1">{{ remainingCount }} items found</p>
```
remained unresolved
2. Changing class on interface remained unclear
3. Conflicts

File: src/app/component/home/models/NewsDto.ts (hunk 2 above)
Comment (nvolodymyr) [reply src/app/component/home/models/NewsDto.ts:5]: I know, that functionality isn't for aot-mode, but it looks like more readable.

Comment (nvolodymyr) [reply src/app/component/home/models/NewsDto.ts]: fixed

File: src/app/component/eco-news/components/remaining-count/remaining-count.component.html (hunk 1 above)
Comment (nvolodymyr) [reply src/app/component/eco-news/components/remaining-count/remaining-count.component.html]: fixed

**Pull request merged**